lbry/lbry/extras/daemon/Daemon.py

```diff
@@ -33,7 +33,8 @@
 from lbry.extras.daemon.ComponentManager import RequiredCondition
 from lbry.extras.daemon.ComponentManager import ComponentManager
 from lbry.extras.daemon.json_response_encoder import JSONResponseEncoder
-from lbry.extras.daemon.comment_client import jsonrpc_post, sign_comment, is_comment_signed_by_channel
+from lbry.extras.daemon.comment_client import jsonrpc_post, sign_comment, sign_abandon_comment
+from lbry.extras.daemon.comment_client import is_comment_signed_by_channel
```

Comment:
either just import `comment_client` as a module and use dot notation to access the functions later in the code:
```
from lbry.extras.daemon import comment_client
```
or split the import list across several lines:
```
from lbry.extras.daemon.comment_client import (
    jsonrpc_post, sign_comment, sign_abandon_comment, is_comment_signed_by_channel
)
```